src/AutorestSwift/View Models/ReturnTypeViewModel.swift

```diff
@@ -36,14 +36,26 @@ struct ReturnTypeViewModel {
     let strategy: String
     let pagingNames: Language.PagingNames?
 
-    init(from response: ResponseViewModel?) {
-        self.strategy = response?.strategy.rawValue ?? ResponseBodyType.noBody.rawValue
+    init(from responses: [ResponseViewModel]?) {
+        var statusCodes = Set<String>()
+        var strategies = Set<ResponseBodyType>()
+        for response in responses ?? [] {
+            strategies.insert(response.strategy)
+            for statusCode in response.statusCodes {
+                statusCodes.insert(statusCode)
+            }
+        }
+
+        assert(strategies.count <= 1, "Different strategy in ResponseViewModel is currently not supported.")
+        self.statusCodes = Array(statusCodes)
+        self.strategy = strategies.first?.rawValue ?? ResponseBodyType.noBody.rawValue
+
+        let response = responses?.first
```

Comment:
If there are multiple responses, what's the rationale for just taking the first one?

Reply:
I want to fix that in a new pr instead of keep growing this one. So existing code only consider the first response, so i leave that as-is

Reply:
Even in your multi-response PR we only consider the first one?

Reply:
In that case, we should leave a TODO to come back here and fix up this logic.

Reply:
and in my multiple-exception PR, it is building code from 'exception' array in code mode, not from multiple response in the code mode.

Reply:
So we still only support 1 response (but with multiple exception) at this point.